README_Template.md

```diff
@@ -1,130 +1,137 @@
+
 This is independent from the framework and can be used wherever - Even in other test frameworks. It is just an assertion library used to assert data is as you expect. It uses an asychronous syntax which may be different to other assertion libraries you may have used.
 
-### TUnit
+## TUnit
+
 This is a helper package to combine the above 3 packages. If you just want a standard test app where you can write, run and assert tests, just install this!
 
-### TUnit.Playwright
-This provides you base classes, similarly to Microsoft.Playwright.NUnit or Microsoft.Playwright.MSTest, to automatically create and dispose of Playwright objects in tests, to make it easier for you to write tests without worrying about lifecycles or disposing. The base classes are named the same as the other libraries: `PageTest`,  `ContextTest`, `BrowserTest`, and `PlaywrightTest`.
+## TUnit.Playwright
 
-## Features
+This provides you base classes, similarly to Microsoft.Playwright.NUnit or Microsoft.Playwright.MSTest, to automatically create and dispose of Playwright objects in tests, to make it easier for you to write tests without worrying about lifecycles or disposing. The base classes are named the same as the other libraries: `PageTest`, `ContextTest`, `BrowserTest`, and `PlaywrightTest`.
+
+# Features
 
 - Native AOT / Trimmed Single File application support
 - Source generated tests
 - Property injection
 - Full async support
 - Parallel by default, with mechanisms to:
-    - Run specific tests completely on their own
-    - Run specific tests not in parallel with other specific tests
-    - Limit the parallel limit on a per-test, class or assembly level
+
+  - Run specific tests completely on their own
+  - Run specific tests not in parallel with other specific tests
+  - Limit the parallel limit on a per-test, class or assembly level
+
 - Tests can depend on other tests to form chains, useful for if one test depends on state from another action. While not recommended for unit tests, this can be useful in integration testing where state matters
 - Easy to read assertions - though you're also free to use whichever assertion library you like
 - Injectable test data via classes, methods, compile-time args, or matrices
 - Hooks before and after: 
-    - TestDiscover
-    - TestSession
-    - Assembly
-    - Class
-    - Test
+
+  - TestDiscover
+  - TestSession
+  - Assembly
+  - Class
+  - Test
+
 - Designed to avoid common pitfalls such as leaky test states
 - Dependency injection support ([See here](https://thomhurst.github.io/TUnit/docs/tutorial-extras/class-constructors))
 - Ability to view and interrogate metadata and results from various assembly/class/test context objects
 
-## Installation
+# Installation
 
 `dotnet add package TUnit --prerelease`
 
-## Example test
+# Example test
 
 ```csharp
-    private static readonly TimeOnly Midnight = TimeOnly.FromTimeSpan(TimeSpan.Zero);
+private static readonly TimeOnly Midnight = TimeOnly.FromTimeSpan(TimeSpan.Zero);
```

Comment:
Inconsistent indentation.

Reply:
I'll take a look. Blame the vscode formatter 😅